packages/magento-product-configurable/components/ConfigurableProductOptions/ConfigurableProductOption.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  useFormAddProductsToCart,
+  AddProductsToCartMutationVariables,
+} from '@graphcommerce/magento-product/components'
+import {
+  nonNullable,
+  filterNonNullableKeys,
+  SectionHeader,
+  ActionCardListForm,
+  ActionCardItemBase,
+} from '@graphcommerce/next-ui'
+import { useWatch } from '@graphcommerce/react-hook-form'
+import { i18n } from '@lingui/core'
+import { Box, SxProps, Theme } from '@mui/material'
+import { ConfigurableOptionsFragment } from '../../graphql/ConfigurableOptions.gql'
+import { useConfigurableOptionsForSelection } from '../../hooks'
+import {
+  ConfigurableOptionValue,
+  ConfigurableOptionValueFragment,
+} from '../ConfigurableOptionValue'
+
+type Props = NonNullable<
+  NonNullable<ConfigurableOptionsFragment['configurable_options']>[number]
+> & {
+  idx: number
+  index: number
```

Comment:
Replace with the type AddToCartItemSelector

Reply:
I'm not sure what you mean, lets look at this together.